editoast/src/main.rs

```diff
@@ -413,15 +412,21 @@ async fn import_railjson(
     args: ImportRailjsonArgs,
     db_pool: Data<DbPool>,
 ) -> Result<(), Box<dyn Error + Send + Sync>> {
-    let railjson_file = File::open(args.railjson_path)?;
+    let railjson_file = match File::open(args.railjson_path.clone()) {
+        Ok(file) => file,
+        Err(e) => {
+            eprintln!("❌ {}: {e}", args.railjson_path.to_string_lossy());
+            exit(1);
+        }
+    };
```

Comment:
We discussed this with @leovalais. We think it's a bit of a waste to lose a test just to have a prettier CLI error.

We've found an alternative, we could add a `CLIError` which is caught by the `main` for a pretty display. When caught the main could `exit(1)` instead of `2` for other types of errors.

What do you think?

Reply:
I agree. Just note that the test I removed does indeed test File::open, not the import_railjson function itself.